src/config/specs/spec.ts

```diff
@@ -0,0 +1,9 @@
+const mocha_jasmine = ["../**/*test.ts"];
```

Comment:
I suppose it's no need to add separate files for specs, suites. As you have src/config/test-runners.ts that also responsible for runner configuration

Reply:
Agree, removes specs folder and updated test-runners file